pipeline/src/helpers/type-guards.ts

```diff
@@ -1,11 +1,15 @@
 import * as prismic from "@prismicio/client";
-import { DataInterface, PrismicImage } from "../types/prismic";
+
+import {
+  DataInterface,
+  PrismicImage,
+} from "@weco/content-pipeline/src/types/prismic";
 
 export function isNotUndefined<T>(val: T | undefined): val is T {
   return typeof val !== "undefined";
 }
 
-function isString(v: any): v is string {
+function isString(v: unknown): v is string {
```

Comment:
Again, I _believe_ this is the valid type in this case, feel free to contradict though!

Reply:
Sounds more accurate in this case 👍